### PR TITLE
Display "unknown" when submission rerecords are 0

### LINKED ISSUE #1572: Display "unknown" when submission rerecords are 0
Old site used to have this logic, and we used it as a reminder for encoders to not put 0 into encode subtitles (because that wouldn't make sense). Currently the parser simply says 0.

### PR DESCRIPTION
**Pull request by vadosnaprimer**
also reset warnings upon submission replacement

fixes #1572

### CODE REVIEW

File: TASVideos.Core/Services/QueueService.cs [hunk 1]
```diff
@@ -400,6 +400,7 @@ public async Task<string> MapParsedResult(IParseResult parseResult, Submission s
 		submission.System = system;
 		submission.CycleCount = parseResult.CycleCount;
 		var warnings = parseResult.Warnings.ToList();
+		submission.Warnings = null;
```
Comment (adelikat) [inline TASVideos.Core/Services/QueueService.cs:403]: what's going on here?

File: TASVideos/Pages/Submissions/View.cshtml [hunk 2]
```diff
@@ -158,7 +158,10 @@
 				<th>@Html.DisplayNameFor(m => m.Submission.Branch)</th>
 				<td>@Model.Submission.Branch</td>
 				<th>@Html.DisplayNameFor(m => m.Submission.RerecordCount)</th>
-				<td>@Model.Submission.RerecordCount</td>
+				<td>@(Model.Submission.RerecordCount > 0
```
Comment (adelikat) [inline TASVideos/Pages/Submissions/View.cshtml:161]: are we sure this is the only place we show a rerecord count on a submission?

File: TASVideos.Core/Services/QueueService.cs (hunk 1 above)
Comment (vadosnaprimer) [reply TASVideos.Core/Services/QueueService.cs:403]: Warnings string was created upon submission creation, and then persisted on updates. I'm clearing it now so the newly parsed movie is the only one involved in warning creation.

File: TASVideos/Pages/Submissions/View.cshtml (hunk 2 above)
Comment (vadosnaprimer) [reply TASVideos/Pages/Submissions/View.cshtml:161]: I think so

**Review by adelikat — APPROVED** [review]

**Pull request merged**